app/models/bishop.rb

```diff
@@ -0,0 +1,5 @@
+class Bishop < Piece
+	def is_obstructed?
```

Comment:
Don't think we want this here, since the Queen also can be obstructed with a diagonal move (thus the shared behavior should be in a method at the Superclass Piece level. I'm okay with the Knight redefinition since that is unique among all the pieces.